src/libs/actions/Account.js

```diff
@@ -2,15 +2,15 @@ import lodashGet from 'lodash.get';
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 import * as API from '../API';
+import NameValuePair from './NameValuePair';
 
 /**
  * Update the account priority mode.
  *
  * @param {String} mode
  */
 function updatePriorityMode(mode) {
-    API.SetNameValuePair({name: 'priorityMode', value: mode});
-    Onyx.set(ONYXKEYS.PRIORITY_MODE, mode);
+    NameValuePair.set('priorityMode', mode, ONYXKEYS.PRIORITY_MODE);
```

Comment:
This method seems like it should be in `NameValuePairs` now and `Account.js` was not one of my best ideas 😅

Reply:
NAB, though, we don't need to reorganize this yet. But maybe as a follow up can get rid of this file and move these methods to `NameValuePairs`.